drivers/modem/gsm_ppp.c

```diff
@@ -559,6 +584,29 @@ static void gsm_configure(struct k_work *work)
 	gsm_finalize_connection(gsm);
 }
 
+void gsm_ppp_start(struct device *device)
+{
+	struct gsm_modem *gsm = device->driver_data;
+	k_delayed_work_init(&gsm->gsm_configure_work, gsm_configure);
+	(void)k_delayed_work_submit(&gsm->gsm_configure_work, K_NO_WAIT);
+}
+
+void gsm_ppp_stop(struct device *device)
+{
+	// FIXME: synchronize with gsm_rx... mutex?
+
+	/* Stop PPP */
+	struct device *ppp_dev = device_get_binding(CONFIG_NET_PPP_DRV_NAME);
+	const struct ppp_api *api;
+
+	if (!ppp_dev) {
+		LOG_ERR("Cannot find PPP %s!", "device");
```

Comment:
It would be useful to print the name of the device here, so something like
`LOG_ERR("Cannot find %s PPP %s!", CONFIG_NET_PPP_DRV_NAME, "device");`